### PR TITLE
Added test to verify comment feature

### LINKED ISSUE #13: Add tests for comment feature

### PR DESCRIPTION
**Pull request by anmol-kumar0815**
Fixes #13

### CODE REVIEW

**Comment by anmol-kumar0815** [comment]
@VarunSriram99 @shreyakurian02 Please review.

File: playwright-tests/e2e/poms/todos.ts [hunk 1]
```diff
@@ -54,4 +62,23 @@ export class TodoPage {
       this.page.getByTestId("tasks-pending-table").getByRole("row").nth(1)
     ).toContainText(newTodo);
   };
+
+  addCommentAndVerifyIt = async ({ comment }: AddCommentProps) => {
```
Comment (VarunSriram99) [inline playwright-tests/e2e/poms/todos.ts]: Let's not add `it` at the end of variable names. This can be `addCommentAndVerify`

**Review by VarunSriram99 — APPROVED** [review]
LGTM

**Comment by anmol-kumar0815** [comment]
I have made changes as per the review. I am merging this PR.

**Pull request merged**